.travis/is-bump-commit.sh

```diff
@@ -2,7 +2,7 @@
 
 isBumpCommit() {
   msg=`git log --pretty=format:'%s' -1`
-  if [ "$msg" = "Automated version bump" ]
+  if [[ "$msg" =~ ^\[pr-bumper\]* ]]
```

Comment:
Could this cause problems if I do something like `git commit -m "fix pr-bumper config"` on a repo using this project?